hyo2/soundspeed/db/export.py

```diff
@@ -250,6 +257,10 @@ def export_profiles_metadata(self, project_name: str, output_folder: str,
             if self.filter_fields.fields['path']:
                 ft.SetField('path', row[5])
 
+            if self.filter_fields.fields['filename']:
+                fn = row[5].replace('/', '\\').split('\\')
```

Comment:
Python has a dedicated package for doing it in a cross-platform way: https://docs.python.org/3.6/library/os.path.html#os.path.basename